dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/enums/TaskTimeoutStrategy.java

```diff
@@ -50,13 +55,11 @@ public String getDescp() {
         return descp;
     }
 
+    private static final Map<Integer, TaskTimeoutStrategy> code2Strategy = Arrays.stream(values()).collect(Collectors.toMap(x -> x.code, x -> x));
+
+
     public static TaskTimeoutStrategy of(int status) {
-        for (TaskTimeoutStrategy es : values()) {
-            if (es.getCode() == status) {
-                return es;
-            }
-        }
-        throw new IllegalArgumentException("invalid status : " + status);
```

Comment:
This is meaningless since this enum only have 3 values. Please open another more meaningful pull request. @LiJie20190102